zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/clock/ClockRecord.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.protocol.impl.record.value.clock;
+
+import io.camunda.zeebe.msgpack.property.LongProperty;
+import io.camunda.zeebe.protocol.impl.record.UnifiedRecordValue;
+import io.camunda.zeebe.protocol.record.value.ClockRecordValue;
+
+public final class ClockRecord extends UnifiedRecordValue implements ClockRecordValue {
+  private final LongProperty pinProperty = new LongProperty("pin", 0);
+  private final LongProperty offsetProperty = new LongProperty("offset", 0);
```

Comment:
Hm, I thought as well, but I figured it would just be confusing, honestly, and having two properties was just more explicit. :thinking: Mostly confusing if we ever had more properties of different types.

But I think that's unlikely, so why not :shrug: